### PR TITLE
Random Walk Metropolis

### PR DESCRIPTION
**Pull request by bdeonovic**
I thought it would be nice to have a very simple sampler implemented for ease of learning how to code up  more complicated ones (See https://github.com/brian-j-smith/Mamba.jl/issues/77)

### CODE REVIEW

**Comment by bdeonovic** [comment]
I added the RWM to the Dyes example and fixed some typos in the docs.

File: doc/examples/dyes.jl [hunk 1]
```diff
@@ -67,6 +67,10 @@ scheme3 = [HMC([:theta], 10.0, 5),
            HMC([:mu], 10.0, 5, eye(dyes[:batches])),
            Slice([:s2_within, :s2_between], [1000.0, 1000.0])]
 
+scheme4 = [RWM([:theta], [50.0]),
+           RWM([:mu], [50.0 for i in 1:dyes[:batches]]),
```
Comment (brian-j-smith) [inline doc/examples/dyes.jl]: Can be shortened with

```julia
fill(50.0, dyes[:batches])
```

File: doc/examples/dyes.jl [hunk 2]
```diff
@@ -80,3 +84,7 @@ describe(sim2)
 setsamplers!(model, scheme3)
 sim3 = mcmc(model, dyes, inits, 10000, burnin=2500, thin=2, chains=2)
 describe(sim3)
+
+setsamplers!(model, scheme3)
```
Comment (brian-j-smith) [inline doc/examples/dyes.jl]: Should use the newly added sampling scheme4:

```julia
setsampler!(model, scheme4)
```

**Comment by bdeonovic** [comment]
Fixed

**Comment by bdeonovic** [comment]
fixed

**Comment by brian-j-smith** [comment]
This is merged into master and will be in the next package release.

**Pull request closed without merging**